db/schema.rb

```diff
@@ -11,18 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141229224941) do
+ActiveRecord::Schema.define(version: 20150306182232) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "active_admin_comments", force: :cascade do |t|
-    t.string   "namespace",     limit: 255
```

Comment:
That's been my experience too. I did go through the whole cycle manually here.

Apropos, given yours and mine observations, I'm very uncertain about advice some people give to delete old migrations.